pkg/kubelet/lifecycle/handlers.go

```diff
@@ -158,6 +158,9 @@ type appArmorAdmitHandler struct {
 }
 
 func (a *appArmorAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult {
+	if attrs != nil && attrs.Pod != nil {
+		klog.InfoS("AppArmor Admit Handler", "pod", klog.KObj(attrs.Pod), "podUID", attrs.Pod.UID)
+	}
```

Comment:
we inconditionally deference both `attrs` and `attrs.Pod` a couple lines below, so you can just do the same; if we want to be defensive, I think it's better done in a separate commit (or PR entirely)